examples/mysql-demo/README.md

```
@@ -0,0 +1,125 @@
+const pending = await orders.find({ fulfilled: false });
+```
+
+### Always Close the Pool
+
+```javascript
+await adapter.close();
+```
+
+## Expected Output
+
+```
```

markdownlint-cli2 (0.22.1)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)